packages/keystone/fields/RegexplessText/index.js

```diff
@@ -0,0 +1,25 @@
+const { Text } = require('@keystonejs/fields')
+
+class RegexplessKnexFieldAdapter extends Text.adapters.knex {
+
+    equalityConditionsInsensitive (dbPath) {
+        return {
+            [`${this.path}_i`]: (value) => (b) => b.whereRaw(`lower("${dbPath.split('.').join('"."')}") = lower(?)`, [value]),
```

Comment:
If value null?

Reply:
looks like it will work as empty string
<img width="366" alt="image" src="https://github.com/open-condo-software/condo/assets/25384290/b0d5ae03-ae37-4cfb-b84d-2c766e7ad39e">